### PR TITLE
`bucket_name` is a missing variable

### PR DESCRIPTION
**Pull request by jingpengwu**
there are two possible reasons:
- authorization failure
- bucket taken.

### CODE REVIEW

File: src/collection.jl [hunk 1]
```diff
@@ -78,7 +78,7 @@ function connect!(store::KeyStore; location::AbstractString="US", empty::Bool=fa
                 error("Unable to create bucket: $(response[:error][:message])")
             end
         elseif code == 403  # forbidden (not available)
-            error("Bucket name already taken: $bucket_name")
+            error("authorization failure or bucket name already taken: $(response[:error][:message])")
```
Comment (joshbode) [inline src/collection.jl]: Error message should start with a capital:

    Authorization failure ...

**Comment by joshbode** [comment]
Good catch - thanks! If you capitalise the error message I'm happy with the change :)

**Comment by jingpengwu** [comment]
Just capitalized it!

**Comment by joshbode** [comment]
Perfect - merging.

**Pull request merged**